samples/cxx/custom/custom.cpp

```diff
@@ -13,8 +13,6 @@
 // This file is part of Cantera. See License.txt in the top-level directory or
 // at https://cantera.org/license.txt for license and copyright information.
 
-#include "cantera/thermo.h"
-#include "cantera/kinetics.h"
```

Comment:
My understanding of the purpose of these headers is as a user-facing import point for the commonly used Cantera classes. This avoids users having to know about the folder structure inside `include/cantera`. I don't think we should remove the use of these.